Apps/Src/Task_PetWDog.c

```diff
@@ -14,7 +14,10 @@ void Task_PetWDog(void *p_arg) {
     // If WDog_BitMap is all set:
     //      Reset watchdog timer
 
-
+     if (WDog_BitMap == 7) {     
```

Comment:
Can you add a comment on why we are checking if it is equal to 7?

Reply:
I added the comment and committed and pushed it but I can't see the pull request option this time. The file is under commits tho.

Reply:
Ok, this part looks good. Once you open a pull request, it will track your branch, so you don't have to reopen it after each commit you make